shuffle/genesis/src/lib.rs

```diff
@@ -69,11 +73,12 @@ pub fn generate_validator_config(
     Ok(validators.pop().unwrap())
 }
 
+/// TODO: remove this code and use Move package manager to build sources when it is available
 pub fn build_move_sources() -> Result<()> {
     // Build the Move code to ensure we get the latest changes in script builders + the genesis WriteSet
     utils::time_it("Building Move code", || {
-        let output = Command::new("df-cli")
-            .args(&["sandbox", "publish"])
+        let output = Command::new("move")
```

Comment:
What's the reason for using move vs df cli?

Reply:
We had to use `df-cli` before because previously we compiled our copy of diem framework modules which use diem native functions. Since we don't have to compile these modules here anymore, we don't need the diem native functions and can just use the simpler move-cli (`move` command) to compile.